test/e2e/nodefeature/nodefeature.go

```diff
@@ -47,6 +47,7 @@ var (
 	PodResources                           = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("PodResources"))
 	ResourceMetrics                        = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("ResourceMetrics"))
 	RuntimeHandler                         = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("RuntimeHandler"))
+	Swap                                   = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("Swap"))
```

Comment:
Can you add a comment explaining what this feature is and which tests should have this label?